.gitignore

```diff
@@ -2,4 +2,5 @@
 .vs/
 .vscode
 src/key-*.xml
-.env
\ No newline at end of file
+.env
+src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/package.json
```

Comment:
curious why this is here? 

It would prevent us from updating the package.json with new dependencies.

Reply:
hmm confused thought you removed line 6 already, still showing as still being here 

<img width="634" height="98" alt="image" src="https://github.com/user-attachments/assets/17495974-e10d-418b-9583-c83feece47cf" />